lib/node_modules/@stdlib/_tools/changelog/generate/lib/main.js

```diff
@@ -315,7 +341,7 @@ function generate( pkg, releaseType ) {
 		}
 	}
 
-	if ( isNamespacePkg ) {
+	if ( isNamespacePkg && !opts.flags ) {
```

Comment:
Why is the `!opts.flags` check necessary here and below?

Reply:
I suggest adding a `format` option rather than an implicit `opts.flags` check used as a proxy.

Reply:
Refactored.